gslib/commands/rsync.py

```diff
@@ -448,6 +448,11 @@
                  what would be copied or deleted without actually doing any
                  copying/deleting.
 
+  -i             This forces rsync to skip any files which exist on the destination
```

Comment:
I think this section is alphabetized. Could you move this between -e and -j?

Reply:
Done